exec/java-exec/src/main/java/org/apache/drill/exec/physical/config/WindowPOP.java

```diff
@@ -33,26 +35,29 @@
   private final NamedExpression[] withins;
   private final NamedExpression[] aggregations;
   private final Order.Ordering[] orderings;
-  private final long start;
-  private final long end;
+  private final boolean rows;
```

Comment:
the SQL reference refers to this as "window frame units". Renaming to frameUnitsRows